docs/FORGER.md

```diff
@@ -100,11 +100,43 @@ This directory will be mounted into the zend container and used to seed the node
     ```
 
 9. **IMPORTANT NOTE**
-- The address  **_"Generated Ethereum Address Key Pair"_** is where rewards will go to. 
-- Rewards are paid to the first ETH address in the wallet of the Forger Node. 
-- **We recommend to not delegate from the node so that no stakes have to be custodied on it, which reduces attack surface.**
-- Stakes should be delegated from web3 wallets like MetaMask. 
-- You can also import this address into MetaMask as an external account so that you can spend the rewards without having to use the node's api.
+   - The address  **_"Generated Ethereum Address Key Pair"_** is where rewards will go to. 
+   - Rewards are paid to the first ETH address in the wallet of the Forger Node. 
```

Comment:
If I well remember, the forger address can be specified in the .conf file and only if it not, the first ETH address in the wallet is used.